qiime2/jupyter/handlers.py

```diff
@@ -0,0 +1,65 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2018, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import os
+import pathlib
+
+import tornado.web as web
+from notebook.base.handlers import IPythonHandler
```

Comment:
This file isn't imported unless `load_jupyter_server_extension` is called, or you import it directly (but don't do that). So I don't think we need to add any of these as actual dependencies. They are optional, and exist if you have Jupyter installed.